tests/bundler.js

```diff
@@ -167,6 +179,39 @@ describe( 'bundler', () => {
 	} );
 } );
 
+describe( 'bundler/jsonPackage', () => {
+	before( () => {
+		rimraf( 'tests/fixtures/jsonPackage/dist' );
+	} );
+
+	after( () => {
+		rimraf( 'tests/fixtures/jsonPackage/dist' );
+	} );
+
+	// #155
+	it( 'should load JSON file', () => {
+
+		function testExecution( filename ) {
+			const codeES5 = readFileSync( filename ).toString();
+			const consoleSpy = spy( console, 'log' );
+
+			// I'm so sorry!
+			eval( codeES5 );
+
+			expect( consoleSpy.calledWithExactly( 'Piotr Kowalski' ) ).to.be.true;
+			expect( consoleSpy.callCount ).to.equal( 1 );
+			consoleSpy.restore();
```

Comment:
Please move this above the assertions. This way the native `console.log` will be restored even if test fails.